packages/web-components/src/components/va-banner/va-banner.tsx

```diff
@@ -49,9 +49,9 @@ export class VaBanner {
 
   /* eslint-disable i18next/no-literal-string */
   /**
-   * The type of the banner. One of 'info', 'error', 'success', 'continue', or 'warning'. This affects both the icon of the AlertBox and the top border color.
+   * The type of the banner. This affects both the icon of the AlertBox and the top border color.
    * */
-  @Prop() type?: string = 'info';
```

Comment:
Changed this prop in the banner component because it gets passed as a status to alert and there were conflicting types between the 'string' definition here and the predefined list of the alert.